.gitlab/lassen-jobs.yml

```diff
@@ -45,36 +45,36 @@ xl_16_1_1_11_gcc_8_3_1:
 # CUDA
 ##########
 
-clang_9_cuda_11_0_2:
+clang_9_cuda_11.1.0:
   variables:
-    SPEC: "+openmp +cuda cuda_arch=70 %clang@9.0.0 ^cuda@11.0.2"
+    SPEC: "+openmp +cuda cuda_arch=70 %clang@9.0.0 ^cuda@11.1.0"
   extends: .build_and_test_on_lassen
```

Comment:
I see that this PR is marked as "draft" so I guess it won’t be merged in the coming release. Is that right?

Trying to assess if this will be merged before or after #1308.

Reply:
@adrienbernede this will be merged for the release. What should the spec be here to not disrupt your CI PR? @mdavis36 we will update compiler specs after the release. Can you match specs in your branch with this https://github.com/LLNL/RAJA/pull/1308 for now?

Reply:
11.0.2 is broken and fails to link device code correctly. Every other version works, I left a comment on #1308 with my request to nix 11.0.2 and bump to 11.1.0.